adapter.js

```diff
@@ -1,207 +1,82 @@
-////////////////////////////////////////////////////////////////////////////
-//
-// Copyright 2016 Realm Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-//
-////////////////////////////////////////////////////////////////////////////
-
-'use strict';
-
+const PostgresAdapter = require('realm-data-adapters').PostgresAdapter
+const Realm = require('realm')
 const fs = require('fs');
 const path = require('path');
-const Realm = require('realm');
-const PostgresAdapter = require('./postgres_adapter.js');
-const PostgresClient = require('libpq');
-
-// CONFIGURABLE
-var access_token = 'ewoJImlkZW50aXR5IjogIl9fYXV0aCIsCgkiYWNjZXNzIjogWyJ1cGxvYWQiLCAiZG93bmxvYWQiLCAibWFuYWdlIl0KfQo=:UH4womynqiP+9/b4kXvGc7E9XrPB3bbdCKJNXhjbWwXN7jbRLlldsG5jCu5CF9vEIYl8nSrORmGpIC/6tztlPzKs3RkIphJWb19uvD3shL6zKy7W+iZrGzPrFDNpEXM4hp4Hq1L/zgxMf1vlElL5JzW2Yq5AybpZZAeP6IVM/xC5bRfCzf7iER8om28WxnVK3CC2buOJbV+dKKWNSVbVl7Ol3N9+Vu2owkByGGGaY0DjzWkTzf+Y+q8r1aoC5MBgOqXhde+bWSiKc2hzFK5FZH1m0KG5+odD40r2mahxdAw304S6ipJcIwGA5pFlBsSNQ/c9K8DYvR0sUZXu9M5xlQ==';
-
-var pg_host = 'localhost';
-var pg_port = 5432;
-var pg_user = 'afish';
-var pg_password = 'password';
-
-// DO NOT CHANGE UNLESS INSTRUCTED
-var pg_dbName = 'dvdrental';
-
-// Create a Postgres client to setup DVD database
-var client = new PostgresClient();
-var connectionString = 'dbname='+pg_user+' host='+pg_host+' port='+pg_port+' user='+pg_user+' password='+pg_password;
-client.connectSync(connectionString);
-
-// HACK - delete admin realm
-try {
-  require( 'child_process' ).execSync('rm -r realm-object-server/adapter/__admin.realm*');
-}
-catch(e) {}
-
-var args = process.argv;
-args.splice(0,2);
-if (args.length > 0) {
-  for (let index in args) {
-    let val = args[index];
-    if (val == '-reset' ||
-        val == '-r') {
-          console.log(`Dropping replication slot '${pg_dbName}'`);
-          client.exec(`SELECT pg_drop_replication_slot('${pg_dbName}');`);
-          console.log(`Removing ${pg_dbName} database`);
-          client.exec(`DROP DATABASE IF EXISTS ${pg_dbName}`);
-          
-          var deleteFolderRecursive = function(path) {
```

Comment:
If you want to reset the state/start over you need to delete all the created realm files.